lib/single-new-eks-cost-monitoring-pattern/grafanaoperatorsecretaddon.ts

```diff
@@ -0,0 +1,102 @@
+import 'source-map-support/register';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import * as eks from "aws-cdk-lib/aws-eks";
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import { Construct } from 'constructs';
+import { createNamespace, dependable } from '@aws-quickstart/eks-blueprints/dist/utils';
```

Comment:
Remoev this file and read from other file in the repo

Reply:
@elamaran11 this does not exist in the repo, i can copy the code over if needed

Reply:
This is the only way to import them, no other option

see
```
lib/single-new-eks-opensource-observability-pattern/istio/istioIngressGatewayAddon.ts

or

lib/single-new-eks-opensource-observability-pattern/istio/istiocniAddon.ts
```